lib/rijksmus-api/client.rb

```diff
@@ -1,11 +1,11 @@
-module Rijksmus 
+module Rijksmus
   module API
     class Client
       include ::HTTParty
 
-      base_uri "https://www.rijksmuseum.nl/api/en"
+      base_uri 'https://www.rijksmuseum.nl/api/en'
       # debug_output
-      default_params :format => "json"
+      default_params format: 'json'
```

Comment:
Why this change? Just curious if this si better syntax and how so :)

Reply:
In this case :format => "json" is very old Ruby syntax.

```ruby
key: 'value'
```
instead:

```ruby
:key => 'value'
```

[Take a look here :)](https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/cop/style/hash_syntax.rb)

Reply:
@nelantone thanks for the links! Really glad to read up on it.